shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModel.kt

```diff
@@ -157,6 +165,7 @@ class FavoritesViewModel(
                         RouteCardData.Context.Favorites,
                         // not depending on now because it only matters for testing
                         now,
+                        location,
```

Comment:
Since setLocation is only called from the favorites page (and not while edit favorites page is open), I think it is safe to pass the latest value of location here so that the sort order on the edit page matches the sort order from nearby.